docs/tutorials/tutorial-custom-domain-with-istio.md

```diff
@@ -148,7 +148,7 @@ spec:
   cloudProfileName: gcp
   kubernetes:
     allowPrivilegedContainers: true
-    version: 1.24.8
+    version: 1.28.2
```

Comment:
You could also update the K8s version in https://github.com/gardener/gardener-extension-shoot-cert-service/blob/709556f67888d61b5977d1e1d6187eacf6ab6ea4/docs/tutorials/tutorial-custom-domain-with-istio.md#L199-L203.